pkg/controllers/vdb/podfacts.go

```diff
@@ -379,12 +380,16 @@ func (p *PodFacts) genGatherScript(vdb *vapi.VerticaDB) string {
 		df --block-size=1 --output=size %s | tail -1
  	`,
 		vdb.GenInstallerIndicatorFileName(),
-		paths.AdminToolsConf,
 		paths.EulaAcceptanceFile,
-		paths.ConfigLogrotatePath,
-		paths.ConfigLogrotatePath,
-		paths.ConfigSharePath,
-		paths.HTTPTLSConfDir, paths.HTTPTLSConfFile,
+		paths.ConfigLogrotatePath, paths.ConfigLogrotatePath,
```

Comment:
Why is each argument passed twice?

Reply:
We test each file like this:
```
echo -n '  %s: '
test -f %s && echo true || echo false
```

We want the file name to be displayed, then the actual test for the file after it. So, we need to reference the same variable twice.

Reply:
Ah Ok, did not see it was `fmt.Sprintf`